### PR TITLE
[FIX] base/account: let users activate/deactivate taxes/currencies fr…

### PR DESCRIPTION
**Pull request by hpr-odoo**
…om list view itself

Task:2155803
Closes:

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account.py [hunk 1]
```diff
@@ -1190,7 +1190,7 @@ class AccountTaxGroup(models.Model):
 class AccountTax(models.Model):
     _name = 'account.tax'
     _description = 'Tax'
-    _order = 'sequence,id'
+    _order = 'active desc, sequence,id'
```
Comment (jco-odoo) [inline addons/account/models/account.py]: @hpr-odoo Was this the case before?

File: addons/web/static/src/scss/list_view.scss [hunk 2]
```diff
@@ -210,7 +210,7 @@
         .o_data_row:not(.o_selected_row) .o_data_cell {
             &.o_toggle_button_cell:not(.o_readonly_modifier) button, // registry: toggle_button
             &.o_boolean_favorite_cell:not(.o_readonly_modifier) .o_favorite, // registry: boolean_favorite
-            .custom-checkbox:not(.o_readonly_modifier) {
+            .custom-checkbox:not(.o_field_widget):not(.o_readonly_modifier) {
```
Comment (jco-odoo) [inline addons/web/static/src/scss/list_view.scss]: @hpr-odoo This should have been solved in v13.0 now, I don't think this is still necessary.  Can you rebase and test?

File: addons/account/models/account.py (hunk 1 above)
Comment (hpr-odoo) [reply addons/account/models/account.py]: @jco-odoo , No it wasn't case before, but it was asked to show active taxes above inactive in spec.

File: addons/web/static/src/scss/list_view.scss (hunk 2 above)
Comment (hpr-odoo) [reply addons/web/static/src/scss/list_view.scss]: sure

**Comment by hpr-odoo** [comment]
@jco-odoo , branch is rebased and it works well.

File: addons/account/models/account.py (hunk 1 above)
Comment (jco-odoo) [reply addons/account/models/account.py]: @hpr-odoo You will have to remove it anyways, because you can not do that in stable

**Comment by hpr-odoo** [comment]
Hi @jco-odoo , i have done it. Thank you :)

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**